src/components/RadioButton/RadioButton.e2e.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'protractor';
```

Comment:
This file can be removed if there aren't any other tests, other than the default one.
The e2e file is part of the generator, because sometimes we need a real DOM for our tests.
But if it's not needed, we can just remove.